.github/workflows/ci.yml

```diff
@@ -269,6 +269,7 @@ jobs:
       - uses: haskell/actions/setup@v1
```

Comment:
Does upgrading to `haskell/actions/setup@v2` (and similarly on line 97, which GitHub won't let me highlight) make a difference?

Reply:
Seems like that worked, thanks! I've made that change in a separate PR #1528.